CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## 18.0.1
```

Comment:
Is this meant to go into 18.1.0? I don't think we usually do verbose descriptions in patch releases.

Maybe keep the changelog entry short ("Added support for rollout strategies to Managed Updates v2") and give a link to the docs for more detailed descripion?

Reply:
In this case should I just put this into the 18.0.0 changelog retroactively?